src/main/java/com/softserve/academy/spaced/repetition/domain/RememberingLevel.java

```diff
@@ -50,4 +50,19 @@ public Integer getNumberOfPostponedDays() {
     public void setNumberOfPostponedDays(Integer numberOfPostponedDays) {
         this.numberOfPostponedDays = numberOfPostponedDays;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
Missing curly brackets. Here twice and twice in the next class.

Last sprint you made a good step ahead - started to write unit test. This sprint I propose to make one more update - install SonarLint plugin for IDEA, it can find issues like this one better than I can.

Reply:
Okay, I'll add them.